.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,49 @@
+name: Benchmark Adaptive Localization
+on:
+  push:
+    branches:
+      - benchmark
+
```

Comment:
So this benchmark branch should be long lived and when we want to benchmark something we push to it?

Reply:
I'll need to change this to `main` before merging.